### PR TITLE
Staging 1.2.0

### PR DESCRIPTION
**Pull request by sastels**
# Description

release candidate 1.2.0

### CODE REVIEW

File: f2/src/AnonymousPage.js [hunk 1]
```diff
@@ -27,14 +28,14 @@ export const AnonymousPage = () => {
               <Lead>
                 <Trans id="anonymousPage.intro" />
               </Lead>
-
+              {console.log(doneForms)}
```
Comment (khalidelaggan) [inline f2/src/AnonymousPage.js:31]: Was this added for testing? should we remove it?

Comment (amazingphilippe) [reply f2/src/AnonymousPage.js:31]: Thought i removed it! Can we just remove it before merging?

Comment (sastels) [reply f2/src/AnonymousPage.js]: whoa, good catch @khalidelaggan ! Removed.

Comment (davidheerema) [reply f2/src/AnonymousPage.js]: was it removed from master as well?

Comment (sastels) [reply f2/src/AnonymousPage.js]: Nope, just this PR

**Pull request merged**